environments/python/Dockerfile

```diff
@@ -2,7 +2,9 @@ FROM alpine
 
 WORKDIR /app
 
-RUN apk add --no-cache python3 python3-dev build-base libev-dev libffi-dev && \
+RUN apk add --update gcc && \
```

Comment:
Sorry, why this change is required? Ideally, the environment image should not have to change at all